tests/dump-custom-resource-classic-apis.yaml

```diff
@@ -0,0 +1,40 @@
+name: Tyk Sync dump Command
```

Comment:
Maybe we can rename the test suite name. for instance, `tyk sync dump cr command`